zeppelin-zengine/pom.xml

```diff
@@ -257,6 +257,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongo-java-driver</artifactId>
+      <version>3.4.1</version>
```

Comment:
@Leemoonsoo Sorry for my late response and thanks for your kind review!

I ran the command and `mongo-java-driver` was leaf node:
```sh
[INFO] \- org.mongodb:mongo-java-driver:jar:3.4.1:compile
```
So, is it correct that I only have to add a license of `mongo-java-driver`?

I just added the license info! Please check 😄 
Thanks